### DISCUSSION TITLE
Moto3 Athena Client: 'get_query_execution' OutputLocation Issue - Filename Missing

### DISCUSSION THREAD

**Discussion by megetron3** [spulec/moto#7419] [Q&A] [answered]
Hey, I'm just using Moto with Athena and expecting `get_query_execution` to return the `OutputLocation` concatenated with the `QueryExecutionId` + ".csv" file.

```
def test_athena():
    query_string = "SELECT * FROM your_table_name"
    output_path = "s3://your-bucket-name/query-results/"

    client = boto3.client("athena")

    response = client.start_query_execution(
        QueryString=query_string,
        QueryExecutionContext={
            'Database': 'your_database_name'
        },
        ResultConfiguration={
            'OutputLocation': 's3://your-bucket-name/query-results/'
        }
    )

    res = client.get_query_execution(
        QueryExecutionId=response["QueryExecutionId"]
    )

    assert res["QueryExecution"]["ResultConfiguration"]["OutputLocation"] == output_path + response[
        "QueryExecutionId"] + ".csv"
```

while getting this error:
```
E       AssertionError: assert 's3://your-bu...uery-results/' == 's3://your-bu...a79c76451.csv'
E         
E         - s3://your-bucket-name/query-results/02635e91-e268-44fd-9696-261a79c76451.csv
E         + s3://your-bucket-name/query-results/

```

do i miss something here?

Additionally, is there a method to simulate the QueryExecutionId to pre-determine the OutputLocation? This approach would facilitate the upload of a dummy S3 file with anticipated results. Such functionality proves valuable when utilizing server/proxy mode alongside with 3rd applications that rely on reading S3 query result files generated by Athena.

**Comment by bblommers** [comment]
Hey @megetron3! The missing `filename.csv` looks like a bug - I'll raise a PR for that now.

Simulating the QueryExecutionId is usually done by seeding Moto. If the seed is the same for every run, then the `random` executionId will also be the same for every run.
This section of the doc explains how to do that: http://docs.getmoto.org/en/latest/docs/configuration/recorder/index.html#deterministic-identifiers

**Reply by bblommers** [reply, marked as answer]
The filename is now returned as part of the `OutputLocation` as of moto >= 5.0.3.dev42

**Reply by megetron3** [reply]
@bblommers many thanks again!
will test it and let you know if the issue solved.
